.github/pull_request_template.md

```diff
@@ -1,3 +1,23 @@
+To ensure full and complete PR descriptions, please use the following template:
+
+Template based on [this article](https://www.pullrequest.com/blog/writing-a-great-pull-request-description/)
+
```

Comment:
@SteveHNH I'm wondering if we can largely eliminate these lines.  I think the purpose of the template is implied based on the sections and explanations provided below.  If the article is useful we could keep the link somewhere in the template, but I sense it may not be needed.